pennylane/utils.py

```diff
@@ -323,6 +326,22 @@ def circuit2():
             + ",".join(string_reps)
         )
 
+    if qml.tape_mode_active():
+        for op in operation_list:
+            try:
+                qml.tape.QueuingContext.remove(op)
```

Comment:
This means that the operation was not recorded in the current tape, correct? Maybe worth having a small comment about it

Reply:
Done!